src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const price = days * PRICE_PER_DAY;
+
+  if (days < SHORT_TERM) {
+    return price;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
There is no need for `days >= SHORT_TERM` condition since the previous if statement ensures that